hera_mc/watch_dog.py

```diff
@@ -48,7 +52,8 @@ def read_forward_list():  # pragma: no cover
             fwd.append(line.strip())
 
 
-def node_verdict(age_out=3800, To=None, testing=False, redishost='redishost'):
+def node_verdict(age_out=3800, To=None, return_as='node_info',
```

Comment:
This new keyword needs to be added to the docstring

Reply:
Done